polaris-cli/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "@shopify/typescript-configs/node-base",
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2019",
```

```suggestion
    "target": "es2020",
```

Suggest bumping to `es2020` for Node.js v14 support (note: This is the [minimum Node.js version required by the Shopify CLI](https://shopify.dev/apps/tools/cli#requirements))